speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DataSelectionMenu.cpp

```diff
@@ -1,16 +1,25 @@
 #include <tgfclient.h>
+#include "guimenu.h"
 #include "legacymenu.h"
 #include "Mediator.h"
 #include "DataSelectionMenu.h"
 #include "ResearcherMenu.h"
 
+#define DATASELECTIONMENU_CAR_DATA                       "dynamic controls/CheckboxCarData"
+#define DATASELECTIONMENU_ENVIRONMENT_DATA          "dynamic controls/CheckboxIndicatorAuditory"
+#define DATASELECTIONMENU_HUMAN_DATA            "dynamic controls/CheckboxUserData"
+#define DATASELECTIONMENU_INTERVENTION_DATA              "dynamic controls/CheckboxInterventionData"
+#define DATASELECTIONMENU_META_DATA            "dynamic controls/CheckboxMetaData"
+
 static void* s_scrHandle  = nullptr;
 static void* s_prevHandle = nullptr;
 static void* s_nextHandle = nullptr;
 
 // Data to store
 tDataToStore m_dataToStore;
 
+int m_dataToStoreGui[5];
```

Comment:
1. Rename to control as that is what it is (return values from CreateControl functions).
2. Add comment to explain what it is
```suggestion
// Control ids for changing GUI elements
int m_dataToStoreControl[5];
```